### ISSUE TITLE
UI gets stuck as display in world

### ISSUE THREAD

**Issue by noiverre** [MykhailoOpryshok/Borukva-Food#6]
![image](https://github.com/user-attachments/assets/64bfca22-7d61-42ae-bbb9-e628d29c8d06)

Client: Polymer, Polydex, PolydexBridge, EMI

**Comment by Stikulzon** [comment]
I'm pretty sure some client mod is causing this (had the same issue before with different mod).

**Comment by noiverre** [comment]
I'll do binary search later, since a player with vanilla client claimed they don't see such visual bug

**Comment by noiverre** [comment]
Did binary search, the culprit is [Chest Tracker](https://modrinth.com/mod/chest-tracker), highly likely the shulker tooltip module but not sure on that and why

**Comment by MykhailoOpryshok** [comment]
This is because the title of the stove and pan has a GUI symbol, and Chest Tracker display this title

**Comment by Stikulzon** [comment]
I don't think that it can be resolved with any Borukva Food modification. Maybe it's possible to add an exception for server-side gui's in the client mod?

**Comment by Stikulzon** [comment]
Other's mod issue
